tests/system/providers/google/suite/example_local_to_drive.py

```diff
@@ -20,22 +20,37 @@
 """
 from __future__ import annotations
 
+import os
 from datetime import datetime
 from pathlib import Path
 
 from airflow import models
 from airflow.providers.google.suite.transfers.local_to_drive import LocalFilesystemToGoogleDriveOperator
 
-SINGLE_FILE_LOCAL_PATHS = [Path("test1")]
-MULTIPLE_FILES_LOCAL_PATHS = [Path("test1"), Path("test2")]
-DRIVE_FOLDER = Path("test-folder")
+DAG_ID = "example_local_to_drive"
+
+FILE_NAME_1 = "test1"
+FILE_NAME_2 = "test2"
+
+LOCAL_PATH = str(Path(__file__).parent / "resources")
+
+SINGLE_FILE_LOCAL_PATHS = [str(Path(LOCAL_PATH) / FILE_NAME_1)]
+MULTIPLE_FILES_LOCAL_PATHS = [str(Path(LOCAL_PATH) / FILE_NAME_1), str(Path(LOCAL_PATH) / FILE_NAME_2)]
+
+DRIVE_FOLDER = "test-folder"
+
+os.environ[
+    "AIRFLOW_CONN_GOOGLE_CLOUD_DEFAULT"
+] = "google-cloud-platform://?extra__google_cloud_platform__scope=https://www.googleapis.com/auth/drive"
```

Comment:
Should be defined in the environment itself (ie breeze config file)

Reply:
ok